test/extended/imageapis/limitrange_admission.go

```diff
@@ -309,7 +313,7 @@ func bumpLimit(oc *exutil.CLI, resourceName kapi.ResourceName, limit string) (ka
 func getMaxImagesBulkImportedPerRepository() (int, error) {
```

Comment:
No. It's in the master's configuration file. The value needs to be passed down to the test.